### DISCUSSION TITLE
Smart EVSE not reading any kWh values

### DISCUSSION THREAD

**Discussion by the-master006** [serkri/SmartEVSE-3#255] [Q&A]
Hi, 
I see only 0 values for the loaded kWh, or for the imported energy and the exported energy. See my screenshots, taken during charing of the car.
I have a setup with 2 smart-evse's both with your latest released firmware, and a sensorbox v2 connected to the ISKRAM DSMR smart meter.
I would expect that it would be able to measure loaded kWh, but apparently it does not. The EV METER values stay 0 kWh.
Or do I per se need a kWh meter connected to it?

My expectation was that since the sensorbox V2 measures the currents from each phase, it could integrate over the time and so on calculate the power per hour. What am I missing?
Thanks!
<img width="299" alt="Schermafbeelding 2024-03-26 210153" src="https://github.com/serkri/SmartEVSE-3/assets/65864740/5e34b533-7290-41c1-bb78-dc8a1dfcc770">
<img width="290" alt="Schermafbeelding 2024-03-26 210239" src="https://github.com/serkri/SmartEVSE-3/assets/65864740/d8b3af73-309a-4d03-b477-356f26336cd4">
<img width="1861" alt="Schermafbeelding 2024-03-26 20572" src="https://github.com/serkri/SmartEVSE-3/assets/65864740/51ccaaf4-dfd8-4986-b291-a2dabdcb3ad1">

**Comment by dingo35** [comment]
No you need an EVmeter for that. The senorbox measures the mains currents, not the EV currents. If you configure an EVMeter on an API you have to, you guessed it, feed it through the API.
